code/modules/vehicles/trolley.dm

```diff
@@ -109,7 +111,9 @@
 		return
 
 	cargo.forceMove(deposit_turf)
-	to_chat(user, span_notice("You unload \the [cargo] from \the [src]."))
+	user.visible_message(
+		span_notice("[user] unloads \the [cargo] from \the [src]."),
+		span_notice("You unload \the [cargo] from \the [src]."))
 	amount_of_cargo--
```

Comment:
Something i learned 2 days ago.

`\the [atom]` is actually completely pointless. Byond will implicitly use `\the` behavior when embedding an atom into a strong